### PR TITLE
[WIP] Logpath calculators

### PR DESCRIPTION
**Pull request by PythonFZ**
- [x] `_post_init_` to convert `model` -> `model.calc` ?

### CODE REVIEW

**Review by M-R-Schaefer — APPROVED** [review]
Much appreciated!

File: ipsuite/calculators/cp2k.py [hunk 1]
```diff
@@ -135,6 +136,31 @@ def atoms(self):
...
+            wfn = self.__dict__["log_path"] / "cp2k-RESTART.wfn"
+            if wfn.exists():
+                # if there is a restart file in the original nwd,
+                # we copy it to the zn.deps node log directory.
+                shutil.copy(wfn, new_path / "cp2k-RESTART.wfn")
+        self.__dict__["log_path"] = value
+
+        with patch(
+            "ase.calculators.cp2k.Popen",
+            wraps=functools.partial(subprocess.Popen, cwd=value),
```
Comment (PythonFZ) [inline ipsuite/calculators/cp2k.py:159]: patch with `self.directory` and do that in a setup function.

Comment (PythonFZ) [reply ipsuite/calculators/cp2k.py:159]: Instead of using `logpath` we could default to a tmp / scratch path and only move the interesting files?

Comment (PythonFZ) [reply ipsuite/calculators/cp2k.py:159]: For `zn.deps` have an `node._origin_` or something like that, which defaults to `self` but is updated for `parent.node = node` to `node._origin_ = parent` ?

**Comment by PythonFZ** [comment]
closed by #83

**Pull request closed without merging**